scripts/Dockerfile

```diff
@@ -211,6 +211,16 @@ RUN apt-get install -y nodejs
 RUN git clone https://github.com/osrf/gzweb.git
 RUN cd gzweb && git checkout gzweb_1.4.0 && npm run deploy --- -m local
 
+# VNC
+RUN apt-get -y install x11vnc
+
+RUN git clone https://github.com/novnc/noVNC.git &&\
```

Comment:
noVNC is officially available through snap. But installing snap also requires systemctl, which is not available in this image. So copying the files from git and checking out the latest release.

Reply:
I found this [another way](https://www.server-world.info/en/note?os=Ubuntu_18.04&p=desktop&f=7) as well, just for reference.

This installs novnc through `apt`, and then we can run the novnc client using `websockify` command.

Reply:
Yep. This seems to be a better way, instead of depending on a particular tag in git.